controllers/managedocs_controller.go

```diff
@@ -1146,6 +1155,12 @@ func (r *ManagedOCSReconciler) reconcileIngressNetworkPolicy() error {
 			return err
 		}
 		desired := templates.NetworkPolicyTemplate.DeepCopy()
+		desired.Spec.Ingress[0].From = []netv1.NetworkPolicyPeer{
+			{
+				PodSelector: &metav1.LabelSelector{},
+			},
+		}
+		desired.Spec.PodSelector = metav1.LabelSelector{}
```

Comment:
Instead of one NetworkPolicy template that we modify in the operator code to satisfy the two Ingress Network policies, I think it would be better for readibility if there were two separate templates. Then for each NetworkPolicy, there would be a single location to check and modify each of the content of the policy.

Reply:
I agree with @renan-campos, It will allow each policy to grow independently in the future if needed, and will make the reconcile code even simpler, as currently, we have no dynamic information in the policies at all.

Reply:
Noted. Have made changes accordingly